aioconsole/server.py

```diff
@@ -16,16 +17,19 @@ def handle_connect(reader, writer, factory, banner=None):
 
 @asyncio.coroutine
 def start_interactive_server(factory=code.AsynchronousConsole,
-                             host='localhost', port=8000, banner=None,
-                             *, loop=None):
+                             host='localhost', port=8000, path=None,
+                             banner=None, *, loop=None):
```

Comment:
The defaults are as they were to provide backwards compatibility with those that don't provide any args (and therefore expect localhost:8000), though the behaviour is admittedly confusing (e.g. providing both sets of args prefers the socket file).

If you're happy requiring `host` and `port` to be set in future versions then that's probably the cleanest way to handle it, and as you've noted it would be consistent with `asyncio.start_server`.

Reply:
I have gone ahead and defaulted all the args to `None`.

Specifying either `host` or `port` will use a network bind with defaults from localhost:8000 (so e.g. `host="192.168.1.1"` will bind to `192.168.1.1:8000`).  Specifying `path` will open a Unix socket.  Providing both or neither will raise a `ValueError`.

Reply:
Great work, thanks !